pkg/apis/kops/cluster.go

```diff
@@ -110,6 +110,8 @@ type ClusterSpec struct {
 	UpdatePolicy *string `json:"updatePolicy,omitempty"`
 	// Additional policies to add for roles
 	AdditionalPolicies *map[string]string `json:"additionalPolicies,omitempty"`
+	// EnableEtcdTLS indicates the etcd service should use TLS between peers and clients
+	EnableEtcdTLS bool `json:"enableEtcdTLS,omitempty"`
```

Comment:
So originally it placed in the EtcdClusterSpec; but issue was the settings effect both cluster regardless since the kube-apiserver would have to setup to use a client cert and TLS on etcd endpoints and the etcd override used by #events/ reuses the same client cert anyhow. Hence why I threw it on the outside, as there is no means for you to set them independently; but clusters are gated by the same option.

If it was placed inside the spec, should it just throw a [validation](https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/validation/legacy.go#L369-L394) error so one is set but the other is not?